samba/data/public/index.html

```diff
@@ -0,0 +1,352 @@
+            color: #63FB72;
+        }
+
+        .text-muted {
+            opacity: 0.8;
+        }
+
+        .text-small {
+            font-size: 14px;
+        }
+
+        hr {
+            width: 100%;
+            height: 2px;
+            background: #2F2C2C;
+            border: none;
+            margin: 40px 0;
+        }
+
+        .container {
+            padding: 40px;
+            max-width: 1440px;
+            margin: auto;
+        }
+
+        .container.terminal-container {
+            background-color: #101010;
+            border-radius: 30px;
+            margin-bottom: 40px;
+        }
+
+        .app-icon {
+            border-radius: 20px;
+        }
+
+        .app {
+            display: flex;
+            margin: 20px 0 40px 0;
+        }
+
+        .app>.app-icon {
+            flex-shrink: 0;
+            height: 140px;
+            width: 140px;
+            box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.95);
+            margin-right: 24px;
+        }
+
+        .app>.app-details>.app-status {
+            display: block;
+            font-size: 20px;
+            margin: 10px 0 0 0;
+        }
+
+        .app>.app-details>.app-name {
+            font-size: 52px;
+            line-height: 52px;
+            font-weight: bold;
+            margin: 10px 0 0 0;
+        }
+
+        #terminal>iframe {
+            border: 0px !important;
+            position: relative !important;
+            height: 70vh !important;
+            width: 100% !important;
+            max-width: 1360px !important;
+        }
+
+        /* Firefox iframe-ception fix */
+
+        body.iframe {
+            background-color: #101010;
+        }
+
+        body.iframe .app-container {
+            display: none;
+        }
+
+        body.iframe .container.terminal-container {
+            border-radius: 0;
+            padding: 0;
+            margin: 0;
+        }
+    </style>
+    <link rel="shortcut icon" type="image/jpg" href="favicon.ico" />
+</head>
+
+<body>
+    <section class="container app-container">
+        <div class="app">
+            <img class="app-icon" src="samba.svg" width="256" height="256" />
+            <div class="app-details">
+                <span class="app-status success">&#9679; Running</span>
+                <h1 class="app-name">Samba</h1>
+            </div>
+        </div>
+        <p class="text-muted">
+            <hr />
+        <h2>How to Connect to the Umbrel SMB Server</h1>
+            <p>Follow these steps to connect to the Umbrel SMB Server:</p>
+
+            <h3>For Windows Users</h3>
+            <ol>
+                <li>Open <strong>File Explorer</strong>.</li>
+                <li>In the address bar, type <code>\\umbrel.local</code> and press <strong>Enter</strong>.</li>
+                <li>When prompted, enter the username: <code>umbrel</code> and the password: <code>changeme</code>.</li>
+                <li>You should now see your <code>/storage</code> directory.</li>
```

Comment:
The Password is the default generated one by umbrelOS. Please adjust it. Also the user doen't know that the folder internally is called `/storage`, maybe we could use something like `You should now see all of your umbrelOS data like downloads, movies, music, etc...`